### ISSUE TITLE
0.28.2 sdist is missing `src/validators/_tld.txt`

### ISSUE THREAD

**Issue by mgorny** [python-validators/validators#380]
When trying to run the tests from 0.28.2 sdist, I'm getting lots of test failures like:

```
FAILED tests/test_domain.py::test_returns_true_on_valid_top_level_domain[example.com-True-False-False] - FileNotFoundError: [Errno 2] No such file or directory: '/tmp/portage/dev-python/validators-0.28.2/work/validators-0.28.2/src/validators/_tld.txt'
```

Apparently the file was there in 0.28.1 but disappeared in 0.28.2.

**Comment by yozachar** [comment]
Hi it was fixed in #379. Please use version [`0.28.3`](https://pypi.org/project/validators/0.28.3/).

**Comment by mgorny** [comment]
Thanks! I can confirm that 0.28.3 looks good.
